### PR TITLE
Fix paths in ABARES STAC

### PR DESCRIPTION
**Pull request by robbibt**
We can't currently access ABARES data on dev or prod:
image

I think this is because some of the STAC paths are wrong. I've attempted to fix these below.

### CODE REVIEW

File: products/others/abares_clum_2020/abares_clum_2020.stac-item.json [hunk 1]
```diff
@@ -374,7 +374,7 @@
       "type": "image/jpeg"
     },
     "alum_class": {
-      "href": "https://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/abares_clum_2020/clum_50m1220m.tiff",
+      "href": "https://dea-public-data.s3-ap-southeast-2.amazonaws.com/projects/abares_clum_2020/clum_50m1220m.tiff",
```
Comment (robbibt) [inline products/others/abares_clum_2020/abares_clum_2020.stac-item.json:377]: Can this just be a relative link instead?

File: products/others/abares_clum_2020/abares_clum_2020.stac-item.json [hunk 2]
```diff
@@ -374,7 +374,7 @@
       "type": "image/jpeg"
     },
     "alum_class": {
-      "href": "https://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/abares_clum_2020/clum_50m1220m.tiff",
+      "href": "clum_50m1220m.tiff",
```
Comment (robbibt) [inline products/others/abares_clum_2020/abares_clum_2020.stac-item.json:377]: Is using a relative link here OK? The GeoTIFF should always live next to the metadata

**Comment by robbibt** [comment]
Ah! Looks like both these changes were already made in the STAC metadata available on S3 here - just not copied back to this repo: https://data.dea.ga.gov.au/projects/abares_clum_2020/abares_clum_2020.stac-item.json

**Review by omad — APPROVED** [review]

**Comment by omad** [comment]
Thanks Robbi. I did some fat fingering too with putting the STAC doc into the wrong path. 

Should be fixed in both environments now.

**Pull request merged**